.github/workflows/examples-esp32.yaml

```diff
@@ -84,3 +85,23 @@ jobs:
 #             - name: Perform CodeQL Analysis
 #               if: ${{ github.event_name == 'push' }}
 #               uses: github/codeql-action/analyze@v1
+            - name: Download Base Binaries
+              uses: dawidd6/action-download-artifact@v2
+              if: ${{ github.event_name == 'pull_request' }}
+              with:
+                  workflow: examples-esp32.yaml
+                  name: esp32-example-build-${{ github.event.pull_request.base.ref }}
```

Comment:
What will be the effect if the base binaries are not available (we are frequently running out of quota so have quite aggressive cleanup scripts)?

Trying to ensure that our CI does not turn red on this.

Reply:
As our master branch is moving forward, new base binaries will be build for every commit. One can rebase his PR on the newest master to get a new image.

This part is not essential, so even it fails, it doesn't cause any trouble.

And I have several other ideas to reduce the usage of artifacts. Let me investigate them later

Reply:
I guess my comment would be translated to:

Do you need a `continue-on-error: true` here?